packages/components/src/organisms/Hero/HeroImage.tsx

```diff
@@ -0,0 +1,38 @@
+import type { HTMLAttributes } from 'react'
+import React, { forwardRef } from 'react'
+
+export interface HeroImageProps extends HTMLAttributes<HTMLDivElement> {
+  /**
+   * Specifies the image URL.
+   */
+  imageSrc: string
+  /**
+   * Alternative description of the image.
+   */
+  imageAlt: string
+  /**
+   * ID to find this component in testing tools (e.g.: cypress, testing library, and jest).
+   */
+  testId?: string
+}
+
+const HeroImage = forwardRef<HTMLDivElement, HeroImageProps>(function HeroImage(
+  { imageAlt, imageSrc, children, testId = 'fs-hero-image', ...otherProps },
+  ref
+) {
+  return (
+    <div ref={ref} data-fs-hero-image data-testid={testId} {...otherProps}>
+      <img
+        data-fs-image
+        loading="eager"
+        src={imageSrc}
+        alt={imageAlt}
+        width={360}
+        height={240}
+        sizes="(max-width: 768px) 70vw, 50vw"
+      />
```

Comment:
this will cause problems with next, because it will recommend using `next/image`.

an alternative is to use something like this:


```
<picture>
  <source srcSet={asset.url} type="image/webp" />
  <img src={asset.url} alt={caption} />
</picture>;
```